.travis.yml

```diff
@@ -13,6 +13,12 @@ services:
 #      - asciidoctor
 before_install:
 - gem install asciidoctor
+# Installing Helm
+- HELM_TGZ=helm-v2.9.1-linux-amd64.tar.gz
+- wget -q https://storage.googleapis.com/kubernetes-helm/${HELM_TGZ}
+- tar xzfv ${HELM_TGZ}
+- PATH=`pwd`/linux-amd64/:$PATH
+- helm init --client-only
```

Comment:
not an Helm expert but doesn't it install just the client but not the Tiller server? Should we test in some way that the chart works so the server is needed?

Reply:
@ppatierno Yes, the `helm init --client-only` was to prepare the CLI chart release management activities like those described in the `Makefile`.  We can add a system test which tests that the generated chart works (which would require us to initialize Tiller on the test cluster).  

I haven't looked at the Strimzi `systemtests` project in detail, but we could also think about making helm the default deployment mechanism (see question 1 in my last comment).  I'll get the systemtests running locally so I can test the feasibility of this.  Do you think that is worth exploring?

Reply:
I'm not quite sure that we want Helm Charts as default deployment mechanism but for sure one of the available mechanisms (so without removing what we have in place today). Regarding the `systemtests` I really think that it's worth exploring having them for helm deployment. @scholzj @tombentley wdyt ?

Reply:
I agree about having an automated test that deploying through helm results in a functioning Strimzi deployment.

Reply:
Ok, I'll add a system test.